src/OrleansRuntime/Streams/PersistentStream/PersistentStreamPullingManager.cs

```diff
@@ -375,17 +389,20 @@ private Task AsyncTimerCallback(object state)
             {
                 foreach (var queueId in queuesToAgentsMap.Keys)
                 {
-                    logger.Info((int) ErrorCode.PersistentStreamPullingManager_PeriodicPrint,
-                        "I am responsible for queue {0} on stream provider {1}. Totally responsible for {0} queues.",
-                        queueId, streamProviderName, queuesToAgentsMap.Count);
+                    Log(ErrorCode.PersistentStreamPullingManager_PeriodicPrint,
+                        "I am responsible for queue {0} on stream provider {1}. Totally responsible for {0} queues.", queueId, streamProviderName, queuesToAgentsMap.Count);
                 }
             }
             else
             {
-                logger.Info((int)ErrorCode.PersistentStreamPullingManager_PeriodicPrint,
-                  "I am responsible for 0 queues on stream provider {0}.", streamProviderName);
+                Log(ErrorCode.PersistentStreamPullingManager_PeriodicPrint, "I am responsible for 0 queues on stream provider {0}.", streamProviderName);
             }
             return TaskDone.Done;
         }
+
+        private void Log(ErrorCode logCode, string format, params object[] args)
```

Comment:
LogInfo - to make it specific that this should not be used for errors or other log levels?